wordpress-mobile/WordPress-iOS
# Experimental page template selector

See https://github.com/wordpress-mobile/gutenberg-mobile/pull/1576

To test:

Launch the app and ensure the template picker is shown only on empty pages and when metro is running (__DEV__ == true)

Update release notes:
